controller/loadBasicPokemonDetails.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+let fs = require('fs');
+let basicPokemonDetail = require('../models/basicPokemonDetail.js');
+let pokemonListPath = require('../resources/json/pokemonlist.json'); 
+
+function loadBasicPokemonDetails() {
+    console.log('Loading Basic Pokemon Details to MongoDB');
+    let data = JSON.stringify(pokemonListPath, function(key,value) {
+        var pokemonID, pokemonName;
+        if(key != 'name' && typeof value.name != 'undefined') {
```

Comment:
@sacdallago using **undefined**, caused one **undefined** entry to be added to the db while **'undefined'** didn't, and I am used to checking undefined this way (just a beginner in javascript)